content/guides/ecommerce-connection/step2-understand-akeneo-pim.md

```diff
@@ -0,0 +1,302 @@
+# Understand Akeneo PIM
+
+## Introduction
+
+### Where can I find the user documentation of Akeneo PIM?
+
+We could spend a lot of time explaining you each PIM's features but there are quite a lot of them... the easiest way is to consult our user documentation that you can find in our [Akeneo Help center](https://help.akeneo.com/pim/index.html).
+
+If you feel that a comprehensive training program would allow you to know everything about Akeneo PIM, we have a complete training program. Do not hesitate to register to one of our [training sessions](https://www.akeneo.com/training/).
+
+### What are Akeneo PIM versions?
+
+First you can consult [this article](https://www.akeneo.com/blog/akeneo-introduces-a-simpler-product-release-cycle/) to understand our PIM release cycle.
+
+Then, in order to understand our latest versions and their functional scope, you can consult our [release notes](https://www.akeneo.com/release-notes/) on our website.
+
+You can also consult our [helpdesk platform](http://helpdesk.akeneo.com/) to know what are our last supported PIM versions or if you don't have access to our Helpdesk platform, please [contact us](https://www.akeneo.com/contact/).
+
+### What are Akeneo PIM editions?
+
+In order to understand our edition and its capabilities, you can consult our [dedicated page](https://www.akeneo.com/compare-editions/) on our website.
+
+### Can I have access to an Akeneo PIM platform?
+
+What could be better than manipulating our PIM to understand its features?
+
+**PIM Community Edition features**
+We have a freely available Akeneo PIM Community Edition demo platform that will allow you to start getting familiar with Akeneo PIM:
+
+[https://demo.akeneo.com](https://demo.akeneo.com)  
+Login/Password: admin / admin
+
+:::warning
+Please note that this Akeneo PIM CE demo instance is for exploratory purposes only and will only allow you to start getting familiar with Akeneo PIM. This instance is reset frequently and is therefore not intended to be used to build a connector.
+:::
+
+**PIM Enterprise Edition features**
+
+If you need an Akeneo PIM Enterprise Edition instance to start developing your connector, please [contact us](https://www.akeneo.com/contact/).
+
+# Understand Akeneo PIM data
```

Comment:
Warning, you shouldn't use a level one title in here.

Reply:
It's an error, I'll fix it.